compiler/rustc_lint_defs/src/lib.rs

```diff
@@ -550,6 +550,9 @@ pub enum BuiltinLintDiagnostics {
         /// The local binding that shadows the glob reexport.
         private_item_span: Span,
     },
+    UnitBindings {
+        span: Span,
+    },
```

Comment:
Is this ever created? I can't find it...

Reply:
(This was for the old bad way of doing lints, but now changed to use the normal way...)